src/StreamService/appsettings.Production.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Serilog": {
+  },
+  "RabbitMQ": {
+    "Host": "lt_do_rabbitmq"
+  },
+  "MemoryCache": {
+    "CacheLiveInMinutes": 120
+  }
```

Comment:
Add CacheLive for Redis

Reply:
Redis was removed from this project for now